docs/samples/rollouts/pinned.yaml

```diff
@@ -6,11 +6,11 @@ spec:
   default: 
     predictor:
       tensorflow:
-        storageUri: "gs://mybucket/mymodel-2"
+        storageUri: "gs://kfserving-samples/models/tensorflow/flowers"
```

Comment:
does it matter which model is being used? I used `flowers` because it already exists from the tensorflow sample, which would then allow `flowers-2` to be used for the canary.

Reply:
It may be more clear to the reader if it's a different model, but it can be exactly the same :). I just mirrored this: ➜  scripts git:(master) ✗ gsutil ls -R gs://kfserving-samples/models/tensorflow/
gs://kfserving-samples/models/tensorflow/:

gs://kfserving-samples/models/tensorflow/flowers-2/:

gs://kfserving-samples/models/tensorflow/flowers-2/0001/:
gs://kfserving-samples/models/tensorflow/flowers-2/0001/saved_model.pb

gs://kfserving-samples/models/tensorflow/flowers-2/0001/variables/:
gs://kfserving-samples/models/tensorflow/flowers-2/0001/variables/variables.data-00000-of-00001
gs://kfserving-samples/models/tensorflow/flowers-2/0001/variables/variables.index

gs://kfserving-samples/models/tensorflow/flowers-2/flowers/:

gs://kfserving-samples/models/tensorflow/flowers-2/flowers/0001/:
gs://kfserving-samples/models/tensorflow/flowers-2/flowers/0001/saved_model.pb

gs://kfserving-samples/models/tensorflow/flowers-2/flowers/0001/variables/:
gs://kfserving-samples/models/tensorflow/flowers-2/flowers/0001/variables/variables.data-00000-of-00001
gs://kfserving-samples/models/tensorflow/flowers-2/flowers/0001/variables/variables.index

gs://kfserving-samples/models/tensorflow/flowers/:

gs://kfserving-samples/models/tensorflow/flowers/0001/:
gs://kfserving-samples/models/tensorflow/flowers/0001/saved_model.pb

gs://kfserving-samples/models/tensorflow/flowers/0001/variables/:
gs://kfserving-samples/models/tensorflow/flowers/0001/variables/variables.data-00000-of-00001
gs://kfserving-samples/models/tensorflow/flowers/0001/variables/variables.index

Reply:
What do you mean by exactly the same? The default and canary predictors are using two similar models right now, but I would assume they're different versions (`flowers` and `flowers-2`). The models being used in this sample are the same ones I saw being used in the tensorflow sample, so that the reader can focus on the canary rollouts feature rather than having to learn what the models are.